ote/src/clj/ote/services/admin.clj

```diff
@@ -577,7 +582,16 @@
        {{:keys [type]} :params
         user :user}
     (require-admin-user "reports/transport-operator" (:user user))
-    (transport-operator-report db type)))
+    (transport-operator-report db type))
+
+
+  ^{:format :csv
+    :filename (str "raportti-" (time/format-date-iso-8601 (time/now)) ".csv")}
+  (GET "/admin/reports/all-ports"
+       {{:keys [type]} :params
+        user :user}
+    (require-admin-user "reports" (:user user))
```

Comment:
Isn't `require-admin-user` nowadays sorta-kinda deprecated for us, should `authorization-fail-response` style design be preferred?
==> rename/refactor also `all-ports` to `all-ports-response` ?

Reply:
Changed